FLEDGE.md

```diff
@@ -211,6 +212,8 @@ The returned `auctionResultPromise` object is _opaque_: it is not possible for a
 
 Optionally, `sellerTimeout` can be specified to restrict the runtime (in milliseconds) of the seller's `scoreAd()` script, and `perBuyerTimeouts` can be specified to restrict the runtime (in milliseconds) of particular buyer's `generateBid()` scripts. If no value is specified for the seller or a particular buyer, a default timeout of 50 ms will be selected. Any timeout higher than 500 ms will be clamped to 500 ms. A key of `'*'` in `perBuyerTimeouts` is used to change the default of unspecified buyers.
 
+Optionally, the `abortSignal` field can be set to an [AbortSignal](https://dom.spec.whatwg.org/#interface-AbortSignal) object to permit aborting the execution of the auction.  When the `abort()` method on the associated `AbortController` is called, an attempt to interrupt the auction will be made. Since the auction executes in parallel to the page, it's possible for this call to happen after the auction actually completed (perhaps unsuccessfully) but before this has been noticed by the caller of `runAdAuction`. In that case, the cancellation attempt is ignored. If the cancellation is successful, the promise is rejected.
```

Comment:
This we need more details here - like how this interacts with setPriority() (Has that even been specced out yet?)

Reply:
... Why would it interact with setPriority?

Reply:
Oh, you mean in terms of updating the priority, not in terms of ordering? Yeah, good chance that doesn't match what I just wrote in the last push, may need to look at fixing that.

Reply:
Yes.  Currently we update the priority immediately (even if the page is navigated, or auction never completes, etc).